content/docs/ui/managing-contacts/segmenting-your-contacts.md

```diff
@@ -80,14 +80,41 @@ Segments can have one Email Activity, Single Send, or Automation condition field
 
 ![The segment builder inside Marketing Campaigns, displaying a new segment with a City condition set to "Denver" and a State, Province, Region condition set to "Colorado".]({{root_url}}/img/create_new_segment_two_fields.png 'Create a new segment with two fields')
 
-Once your segment is processed, you will see a count of all contacts who meet your segment conditions, as well as a sample of these 50 contacts. To see all contacts included in your segment, click "Export CSV" to download a complete file. 
+Once your segment is processed, you will see a count of all contacts who meet your segment conditions, as well as a sample of these 50 contacts. To see all contacts included in your segment, click "Export CSV" to download a complete file.
 
 <call-out type="warning">
 
 Segments built using engagement data such as "was sent" or "clicked" will take approximately 30 minutes to begin populating.
 
 </call-out>
 
+## Segment refresh cadence
+
+Contacts will be added to or removed from your segment as they meet or no longer meet the conditions of your segment. This can be driven by:
+
+- New contacts being added to or updated within Marketing Campaigns, whether via CSV upload, API integration, Signup Forms, or manual changes.
+- Contact engagement behavior, such as opening or clicking an email in a way that’s relevant to your segment conditions.
+
+Typically, contacts that meet your criteria will appear in your segment within 15 minutes. You can see all contacts who are included in the segment at any given time by clicking **Export CSV**. To check for a specific contact, you can search by the contact’s email address.
+
+<call-out type="warning">
+
+Segments built using engagement data such as "was sent" or "clicked" will take approximately 30 minutes to begin populating.
+
+</call-out>
+
+Segment samples and counts are refreshed approximately once per hour; they do not update immediately. If there are no changes to a segment since the last refresh, the sample and UI count displayed will be refreshed at increasing time intervals with a maximum sample and count refresh delay of 24 hours.
```

Comment:
Change "if there are no changes to a segment" to "if no contacts are added or removed from the segment since..."